resources/manage.rb

```diff
@@ -91,7 +91,7 @@
       ssh_keys = []
       if u['ssh_keys']
         Array(u['ssh_keys']).each do |key|
-          if key.start_with?('https')
+          if key.start_with?('http')
```

Comment:
we talked about this at the sous-chefs meeting on Jul 22 2019, the concern is MITM and if we want to support use cases pulling over http I think it needs to be a `curl --insecure` type setup.